### DISCUSSION TITLE
Question about " air circulation external" macro.

### DISCUSSION THREAD

**Discussion by AlexySvider** [xblax/flashforge_ad5m_klipper_mod#230] [Q&A]
When I'm turning it on the inner circulation fan turns on too, is that supposed to happen?
because if you turn on the inner circulation only the inner fan turns on.

**Comment by consp** [comment]
Both fans runs with external, only the internal runs with internal. There is a duct in the system which redirects the flow outside for the "internal" fan if you set it to external.
